README.md

```diff
@@ -16,20 +16,27 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Write a 1-2 sentence summary of what data the pipeline is for and what it does -->
-**nf-core/hicar** is a bioinformatics best-practice analysis pipeline for This pipeline analyses data for HiCAR data, a robust and sensitive multi-omic co-assay for simultaneous measurement of transcriptome, chromatin accessibility and cis-regulatory chromatin contacts..
+**nf-core/hicar** is a bioinformatics best-practice analysis pipeline for HiC on Accessible Regulatory DNA (HiCAR) data, a robust and sensitive assay for simultaneous measurement of chromatin accessibility and cis-regulatory chromatin contacts. Unlike the immunoprecipitation-based methods such as HiChIP, PlAC-seq and ChIA-PET, HiCAR does not need antibody available. HiCAR utilize the Assay of Transposase-Acessible Chromatin to anchor the chromatin interactions. HiCAR is a feasible tool to study the chromatin interactions for low input samples nad the samples with no available antibodies.  
```

Comment:
https://github.com/nf-core/hicar/pull/1#discussion_r741469407

Thanks for the improvement here 👍🏻  Are there any citations for this method? I guess https://www.biorxiv.org/content/10.1101/2020.11.02.366062v1 ?

Reply:
Yes. It is. I will update that.